xmtp/src/storage/unencrypted_store/models.rs

```diff
@@ -0,0 +1,53 @@
+use std::time::{SystemTime, UNIX_EPOCH};
+
+use super::schema::messages;
+use diesel::prelude::*;
+
+/// Placeholder type for messages returned from the Store.
+#[derive(Queryable, Debug)]
+pub struct DecryptedMessage {
+    pub id: i32,
+    pub created_at: i64,
+    pub convoid: String,
+    pub addr_from: String,
+    pub content: String,
+}
+
+/// Placeholder type for messages being inserted into the store. This type is the same as
+/// DecryptedMessage expect it does not have an `id` feild. The field is generated by the
+/// store when it is inserted.
+#[derive(Insertable, Clone, PartialEq, Debug)]
+#[diesel(table_name = messages)]
+pub struct NewDecryptedMessage {
+    pub created_at: i64,
+    pub convoid: String,
+    pub addr_from: String,
+    pub content: String,
```

Comment:
We know the `content` is going to be `bytes` eventually to support content types. Should we just start there and save ourselves a migration later.